docs/source/theming/custom-styling.md

```diff
@@ -11,6 +11,9 @@ myst:
 
 ## Semantic UI
 
+```{include} /_inc/_semanticui-deprecation.md
```

Comment:
Always check your local build, or at least verify on RTD previews https://volto--6541.org.readthedocs.build/theming/custom-styling.html.

All includes _must_ be relative to the current file, not the documentation root, else it will break in the main documentation. Please update the others.

````suggestion
```{include} ../_inc/_semantic-ui-deprecation.md
````